code/game/jobs/job/civilians/other/reporter.dm

```diff
@@ -0,0 +1,39 @@
+#define MILITARY_VARIANT "Mil"
+#define CIVILIAN_VARIANT "Civ"
+
+/datum/job/civilian/reporter
+	title = JOB_COMBAT_REPORTER
+	total_positions = 1
+	spawn_positions = 1
+	selection_class = "job_cl"
+	supervisors = "the acting commanding officer"
+	gear_preset = /datum/equipment_preset/uscm_ship/reporter
+	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
+	selection_class = "job_cl"
+
+	job_options = list(CIVILIAN_VARIANT, MILITARY_VARIANT)
+
+	var/military = FALSE
+
+/datum/job/civilian/reporter/handle_job_options(option)
+	if(option != job_options[1])
```

Comment:
hard coding based on the position in the list isn't ideal, if option == CIVILIAN_VARIANT would be fine (or associating the list of job options with gear presets or w/e)